packages/openapi-to-graphql/src/index.ts

```diff
@@ -491,11 +586,45 @@ function getFieldForOperation(
      */
     requestPayloadDef: operation.payloadDefinition,
     parameters: operation.parameters,
-
     operation,
     data
   })
 
+  if (operation.isSubscription) {
+    const responseSchemaName = operation.responseDefinition
+      ? operation.responseDefinition.graphQLInputObjectTypeName
```

Comment:
I believe this should use `graphQLObjectTypeName` instead of `graphQLInputObjectTypeName`.

Reply:
Yes it would make sense, probably a copy/paste mistake.